extension.js

```diff
@@ -206,7 +207,7 @@ const BingWallpaperIndicator = new Lang.Class({
         } else {
             let message = this.explanation;
             if (this.copyright != "")
-                message += "\n" + this.copyright + ""
+                message += "\n"+""+ this.copyright+ "\n"+BingImageURL+""
```

Comment:
What is the purpose of the empty strings after the "\n" and at the end of the line?

Reply:
Probably a new line that was later deleted. I don't like the url being included in the notification here, can you explain this to me @gwdgithubnom ? I presume this is just for debugging purposes?

Reply:
To print the variable of BingImageURL, it is just for the user could get the description of image URL when he wants to get. It may look not very good, so complete this function by set a link.  like <a></a>.

Reply:
Ok, what we can do instead is create a menu option to copy the image location.